openpype/pipeline/publish/lib.py

```diff
@@ -609,8 +615,17 @@ def context_plugin_should_run(plugin, context):
 def get_instance_staging_dir(instance):
     """Unified way how staging dir is stored and created on instances.
 
-    First check if 'stagingDir' is already set in instance data. If there is
-    not create new in tempdir.
+    First check if 'stagingDir' is already set in instance data.
+    In case there already is new tempdir will not be created.
+
+    It also supports `OPENPYPE_TMPDIR`, so studio can define own temp
+    shared repository per project or even per more granular context.
+    Template formating is supported also with optional keys. Folder is
+    created in case it doesnt exists.
+
+    Available anatomy formatting keys:
+        - root[work | <root name key>]
+        - project[name | code]
 
     Note:
         Staging dir does not have to be necessarily in tempdir so be carefull
```

Comment:
```suggestion
        Staging dir does not have to be necessarily in tempdir so be careful
```

Reply:
also the line below shoud be
```
about its use.
```
as it is possesive form. Can't add suggestions on the code not affected by this PR though :)